### DISCUSSION TITLE
Source of data for spontaneousReactions.tsv in HumanGEMAdapter.m

### DISCUSSION THREAD

**Discussion by PkiwiBird** [SysBioChalmers/GECKO#370] [Q&A]
In the model adapter, there is a function to get a logical with spontaneous reactions `spont` and names of reactions that are spontaneous `spontRxnNames`. Some consider spontaneous reactions not catalysed by enzymes, while others consider those that release Gibbs energy. Are you in this case refering to the first definition? If so, why MAR04740 in human Atlas [has an EC number associated to it](https://metabolicatlas.org/explore/Human-GEM/gem-browser/reaction/MAR04740), while it is considered spontaneous in [`spontaneousReactions.tsv` ](https://github.com/SysBioChalmers/GECKO/blob/main/tutorials/light_ecModel/data/spontaneousReactions.tsv) file?

**Comment by edkerk** [comment]
It is the first definition: reactions that occur without requiring any enzyme (and they will therefore not have any protein cost). This is used to when running [`getStandardKcat`](https://sysbiochalmers.github.io/GECKO/doc/src/geckomat/gather_kcats/getStandardKcat.html)

Why MAR04740 has an E.C. number associated, while at the same time considered as spontaneous, is a matter of [`human-GEM`](https://github.com/SysBioChalmers/Human-GEM/discussions), as the `spontaneousReactions.tsv` was extracted from the [`reactions.tsv`](https://github.com/SysBioChalmers/Human-GEM/blob/main/model/reactions.tsv) file in that repo (I will add a comment in the model adapter of the `light_ecModel` tutorial, as this is currently not documented).

I'm not familiar with MAR04740, identifying it as spontaneous might be a mistake. However, it does not have to be, 6-phosphogluconolactonase is often considered as an enzyme that can catalyze a reaction that can also occur spontaneously.
